### DISCUSSION TITLE
Lucene query where field length is greater than

### DISCUSSION THREAD

**Discussion by devlife** [OrchardCMS/OrchardCore#13451] [Q&A]
I'm trying to run a Lucene query where HtmlBodyPart length is greater than 32000 but I just can't figure it out.
What am I doing wrong?

```
{
  "query": {
    "term": {
      "BlogPost.HtmlBody.length": {
      	"gt": 10
      }
    }
  }
}
```

That returns the following error:

![image](https://user-images.githubusercontent.com/237369/226490646-489b5f0a-57fe-42a6-9107-cd3d54d1d90a.png)

**Comment by devlife** [comment]
Anybody able to help with this?

**Comment by MikeAlhayek** [comment]
I never tried this, but you may want to try `BlogPost.HtmlBodyPart.Html.Length`

**Reply by devlife** [reply]
Adding `.Html` removes the error but returns zero records 🤔

**Reply by MikeAlhayek** [reply]
Try something like this instead 

{
  "query": {
    "filter": "Length(BlogPost.HtmlBodyPart.Html) gt 6"
  }
}

**Reply by devlife** [reply]
That's returning `Invalid query : Unable to cast object of type 'Newtonsoft.Json.Linq.JValue' to type 'Newtonsoft.Json.Linq.JObject'.`.

**Reply by ns8482e** [reply]
I do t think Length filter is implemented

**Reply by devlife** [reply]
That would make sense!
